datasets/gov/kapines/velionys.csv

```diff
@@ -755,4 +755,31 @@ id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri
 ,,,,,eile,string,,,,4,open,,"Velionio kapavietės eilė kapinėse, jei ji egzistuoja (ypač naudojama tvarkingo išplanavimo ir naujose kapinėse). Jeigu kapinėse nebuvo naudojamos eilės, tuomet žymima 000.",
 ,,,,,kapaviete,string,,,,4,open,,Velionio kapavietės numeris kapinėse,
 ,,,,,vieta_kape,integer,,,,4,open,,"Velionio palaidojimo vieta kapavietėje. Pavyzdžiui, 3 vietų kapavietėje per vidurį laidojamam velioniui būtų žymima 2.",
-,,,,,geometrija,"geometry(3857)",,,,4,open,,"Velionio kapavietės vietos taško koordinatės pagal WGS84 / Pseudo-Mercator koordinačių sistemą.",
+,,,,,geometrija,geometry(3857),,,,4,open,,Velionio kapavietės vietos taško koordinatės pagal WGS84 / Pseudo-Mercator koordinačių sistemą.,
+,,,,UkmergesRajonas,,,vda_id,,,4,,,Ukmergės rajono savivaldybės kapinėse palaidotų velionių duomenys,
+,,,,,vda_id,string,,,,4,open,,Valstybės Duomenų Agentūros įrašo identifikatorius,
+,,,,,savivaldybe,string,,,,4,open,,Savivaldybės pavadinimas,
+,,,,,kapo_id,ref,/datasets/gov/kapines/registras/kapavietes/UkmergesRajonas[kapo_id],,,4,open,,Kapavietės identifikatorius,
+,,,,,ar_neatpazintas,boolean,,,,0,open,,Neatpažinto velionio požymis,
+,,,,,ar_perlaidotas,boolean,,,,4,open,,"Požymis, ar velionis buvo perlaidotas",
+,,,,,ar_urnoje,boolean,,,,4,open,,"Požymis, ar velionis palaidotas urnoje",
+,,,,,vardas,string,,,,4,open,,Velionio vardas,
+,,,,,pavarde,string,,,,4,open,,Velionio pavardė,
+,,,,,lytis,string,,,,4,open,,"Velionio lytis, išskaičiuota iš asmens kodo.",
+,,,,,ar_izymybe,boolean,,,,0,open,,"Požymis, ar velionis buvo žinomas asmuo",
```

Comment:
Brandos lygis 0?

Reply:
Labas. Brandos lygis nurodytas 0, kadangi šie stulpeliai nėra kaupiami LIKSGIS kapinių sistemoje, iš kurios bus atverinėjamas šis rinkinys. Pagal ADP dokumentaciją tokie stulpeliai žymimi 0.
![image](https://github.com/user-attachments/assets/595b2f1c-f0bc-4166-ac4b-d59fd8527ebc) 
Šiuos tuščius stulpelius įtraukiau, kad visos rinkinio lentelės būtų unifikuotos. Aišku, tokie stulpeliai numuš bendrą rinkinio brandos lygį.. Vienintelė išeitis būtų išmesti tokius stulpelius iš ADSA, bet tada rinkiniai jau skirsis pagal savo stulpelių struktūrą..

Reply:
Supratau, be dokumentacijos po ranka žiūrėjau, tai pamiršau apie šitą atvejį. 
Na, numuš, bet bendra struktūra tarp skirtingų savivaldybių turbūt prioritetas.

Reply:
Va va, dėl to ir galvojau, kad vienodą struktūrą visose savivaldybėse būtų svarbu išlaikyti